docs/membership-policy.md

```diff
@@ -0,0 +1,29 @@
+# CI maintainers
+
+This document outlines the various responsibilities of the maintainer role in
+KubeVirt CI.
+
+KubeVirt CI maintainers are community members that have access to CI
+infrastructure.
+
+## Requirements
+
+* [Approvers] of https://github.com/kubevirt/project-infra for at least 1 month.
+
+* Nominated by a CI maintainer.
```

Comment:
```suggestion
* Nominated by a current CI maintainer.
  - fgimenez
  - rmohr
  - dhiller
```

Reply:
Or maybe list them elsewhere, in a machine-readable format?

Reply:
Yep, much better, I added an `OWNERS_ALIASES` file with a `ci-maintainer` alias according to https://www.kubernetes.dev/docs/guide/owners/#owners_aliases